core/controllers/contributor_dashboard_admin.py

```diff
@@ -755,3 +781,94 @@ def get(self) -> None:
             'question_reviewers_count': community_stats.question_reviewer_count
         }
         self.render_json(response)
+
+
+def get_translation_coordinator_frontend_dict(
+    backend_stats: List[suggestion_registry.TranslationCoordinatorStats]
+) -> List[suggestion_registry.TranslationCoordinatorStatsDict]:
+    """Returns corresponding stats dicts with all the necessary
+    information for the frontend.
+
+    Args:
+        backend_stats: list. TranslationCoordinatorStats domain object.
+
+    Returns:
+        list. Dict representations of TranslationCoordinatorStats
+        domain objects with additional keys:
+            translators_count: int.
+            reviewers_count: int.
+    """
+    stats_dicts = [
+        stats.to_dict() for stats in backend_stats
+    ]
+
+    for stats_dict in stats_dicts:
+        coordinator_activity_list = []
+        # Here we use MyPy ignore because MyPy doesn't allow key addition
+        # to TypedDict.
+        stats_dict['translators_count'] = ( # type: ignore[misc]
+            contribution_stats_services.get_translator_counts(
+                stats_dict['language_id']))
```

Comment:
The typed dict we are getting in param are the values from backend but we are returning the dict with some additional values. I have implemented 2 different typed dicts for param and return. The reason for using type ignore is because if we add a key in a typed dict mypy considers it an error. I have mentioned the same in above comment.